opl/generators/notifications_ingress.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os.path
+import random
+import jinja2
+import jinja2.meta
+
+import opl.gen
+import opl.generators.packages
+import opl.generators.generic
+import generic
+
+
+class EgressHostsGenerator(generic.GenericGenerator):
```

Comment:
Please call the generator something else. Like `NotificationsMessagesGenerator` or so.

Reply:
Hi @jhutar did this change :)